hardware/products/ucm/intro.md

```diff
@@ -28,20 +27,56 @@ The peripherals found on each module may vary.
 | VBAT (Battery Backup for RTC)                         | 1         |
 | JTAG                                                  | 1         |
 
+> ### Peripherals explained
+> #### UART (Universal Asynchronous Receiver/Transmitter) -
+> #####UART is an asynchronous serial communication method to allow the user the ability to configure the speed and format of the data sent. [*Example TinyCLR code*](../../../tinyclr/tutorials/uart.md)
```

Comment:
No header for the interface description, just start typing.

Reply:
I personally like it with the smaller text size but two other people were asked their opinion and I decided I will remove the header without argument.